src/annotator/config/test/settings-test.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
-var settings = require('../settings');
+var settingsFrom = require('../settings');
```

Comment:
For consistency, it would be preferable to make modules which default-export a single value have names which match the export name, ie. `settings-from.js`.